### ISSUE TITLE
eval failing with `RangeError: Invalid count value: -10`

### ISSUE THREAD

**Issue by bellini666** [promptfoo/promptfoo#1028]
Just got this in our CI

```
/home/circleci/codriver/tools/promptfoo/node_modules/promptfoo/dist/src/updates.js:34
        const border = '='.repeat(process.stdout.columns - 10);
                           ^

RangeError: Invalid count value: -10
    at String.repeat (<anonymous>)
    at checkForUpdates (/home/circleci/codriver/tools/promptfoo/node_modules/promptfoo/dist/src/updates.js:34:28)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async main (/home/circleci/codriver/tools/promptfoo/node_modules/promptfoo/dist/src/main.js:185:5)
```

The strangest thing is that:

1) I can't reproduce the same issue at all locally
2) It was working correctly on CI before, started failing a couple of hours ago without _any changes_ to any dependencies (it is using `npx promptfoo`, but we have it installed on `node_packages` at version `0.66.0`) and neither to any of the config/assertions

**Comment by typpo** [comment]
Thanks for flagging.  This code path is checking for updates and outputting a banner if there is a new version.  0.67.0 was _just_ released, so that's why the problem just started :)

As a temp workaround, set the environment variable `PROMPTFOO_DISABLE_UPDATE=1`.  This will prevent it from checking for updates.

Alternatively, updating CI to use promptfoo 0.67.0 will fix the problem as well.

I will include a fix in the next version.

**Comment by mldangelo** [comment]
A fix for this was deployed in [0.68.0](https://github.com/promptfoo/promptfoo/releases/tag/0.68.0) Please let us know if you run into any other issues and thanks for reporting it!
